### PR TITLE
Clean flake8 lint warnings

### PR DESCRIPTION
**Pull request by janiversen**
Removed the remaining disabled codes:
   C416, D200, D205, D209, D400, D401, NQA101
in first commit.

second commit is source code changes due to 1).

### CODE REVIEW

File: pytradfri/device/__init__.py [hunk 1]
```diff
@@ -152,14 +152,14 @@ def power_source(self):
 
     @property
     def power_source_str(self):
-        """String representation of current power source."""
+        """Representat current power source."""
```
Comment (MartinHjelmare) [inline pytradfri/device/__init__.py:155]:
```suggestion
        """Represent current power source."""
```

File: pytradfri/gateway.py [hunk 2]
```diff
@@ -195,29 +194,29 @@ def process_result(result):
...
 
         return Command("post", [ROOT_GATEWAY, ATTR_GATEWAY_REBOOT])
 
     def set_commissioning_timeout(self, timeout):
-        """Put the gateway in a state in which it accepts pairings from
-        switches, dimmers and motion sensors for up to timeout seconds.
+        """Put the gateway in pairing state.
 
-        Returns a Command."""
+        a state in which it accepts pairings from
```
Comment (MartinHjelmare) [inline pytradfri/gateway.py:207]:
```suggestion
        The pairing state is when the gateway accepts pairings from
```

File: pytradfri/gateway.py [hunk 3]
```diff
@@ -228,12 +227,12 @@ class GatewayInfo:
...
 
     def __init__(self, raw):
-        """Setup object of class."""
+        """Create object of class."""
         self.raw = raw
 
     @property
     def id(self):
-        """This looks like a value representing an id."""
+        """Look like a value representing an id."""
```
Comment (MartinHjelmare) [inline pytradfri/gateway.py:235]:
```suggestion
        """Return the gateway id."""
```

File: pytradfri/gateway.py [hunk 4]
```diff
@@ -260,7 +259,7 @@ def current_time_iso8601(self):
 
     @property
     def first_setup(self):
-        """This is a guess of the meaning of this value."""
+        """Guess the meaning of this value."""
```
Comment (MartinHjelmare) [inline pytradfri/gateway.py:262]:
```suggestion
        """Return the time when gateway was first set up."""
```

**Comment by janiversen** [comment]
Thanks for your text changes, most of these was actually the original text, but your suggestions are better !

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**